src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Manufacturer/Blocks/form.html.twig

```diff
@@ -0,0 +1,119 @@
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(manufacturerForm) }}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">star</i>
+    {{ 'Brands'|trans({}, 'Admin.Catalog.Feature') }}
+  </h3>
+  <div class="card-block row">
+    <div class="card-text">
+      {% set invalidCatalogNameHint %}
+        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
+      {% endset %}
+
+      {{ ps.form_group_row(manufacturerForm.name, {}, {
+        'label': 'Name'|trans({}, 'Admin.Global'),
+        'help': invalidCatalogNameHint
+      }) }}
+
+      {% set invalidGenericNameHint %}
+        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>={}' }}
+      {% endset %}
+
+      {{ ps.form_group_row(manufacturerForm.short_description, {}, {
+        'label': 'Short description'|trans({}, 'Admin.Global'),
+      }) }}
+
+      {{ ps.form_group_row(manufacturerForm.description, {}, {
+        'label': 'Description'|trans({}, 'Admin.Global')
+      }) }}
+
+      {% set logoHint %}
+        {{ 'Upload a brand logo from your computer.'|trans({}, 'Admin.Catalog.Help') }}
+      {% endset %}
+
+      {{ ps.form_group_row(manufacturerForm.logo, {}, {
+        'label': 'Logo'|trans({}, 'Admin.Global'),
+        'help': logoHint
+      }) }}
+
+      {% if logoImage is defined and logoImage is not null %}
+        <div class="form-group row">
+          <label for="" class="form-control-label"></label>
+          <div class="col-sm">
+            {% include '@PrestaShop/Admin/Sell/Catalog/Manufacturer/logo_image.html.twig' %}
+          </div>
+        </div>
+      {% endif %}
+
+      {{ ps.form_group_row(manufacturerForm.meta_title, {}, {
+        'label': 'Meta title'|trans({}, 'Admin.Global'),
+        'help': invalidGenericNameHint
+      }) }}
+
+      {{ ps.form_group_row(manufacturerForm.meta_description, {}, {
+        'label': 'Meta description'|trans({}, 'Admin.Global'),
+        'help': invalidGenericNameHint
+      }) }}
+
+      {% set keywordHint %}
+        {{ 'To add "tags," click inside the field, write something, and then press "Enter."'|trans({}, 'Admin.Catalog.Help') }}
```

Comment:
Localization is fine here, but the source string needs to be reworked a bit, quotation marks are badly placed: _To add "tags", click inside the field, write something, and then press "Enter"._

Reply:
@zuk3975, it has to be modified at the end of the string as well.